README.md

```diff
@@ -185,101 +200,279 @@ if ($launch->has_ags()) {
+$dl->outputResponseForm([$resource]);
 ```
 
 Alternatively you can just request the signed JWT that will need posting back to the platform by calling.
+
 ```php
-$dl->get_response_jwt([$resource]);
+$dl->getResponseJwt([$resource]);
 ```
 
 ## Calling Services
+
 ### Names and Roles Service
 
 Before using names and roles you should check that you have access to it.
+
 ```php
-if (!$launch->has_nrps()) {
+if (!$launch->hasNrps()) {
     throw new Exception("Don't have names and roles!");
 }
 ```
 
 Once we know we can access it, we can get an instance of the service from the launch.
+
 ```php
-$nrps = $launch->get_nrps();
+$nrps = $launch->getNrps();
 ```
 
 From the service we can get an array of all the members by calling:
+
 ```php
-$members = $nrps->get_members();
+$members = $nrps->getMembers();
 ```
 
 ### Assignments and Grades Service
+
 Before using assignments and grades you should check that you have access to it.
+
 ```php
-if (!$launch->has_ags()) {
+if (!$launch->hasAgs()) {
     throw new Exception("Don't have assignments and grades!");
 }
 ```
 
 Once we know we can access it, we can get an instance of the service from the launch.
+
 ```php
-$ags = $launch->get_ags();
+$ags = $launch->getAgs();
 ```
 
-To pass a grade back to the platform, you will need to create an `LTI\LTI_Grade` object and populate it with the necessary information.
+To pass a grade back to the platform, you will need to create an `Packback\Lti1p3\LtiGrade` object and populate it with the necessary information.
+
 ```php
-$grade = LTI\LTI_Grade::new()
-    ->set_score_given($grade)
-    ->set_score_maximum(100)
-    ->set_timestamp(date(DateTime::ISO8601))
-    ->set_activity_progress('Completed')
-    ->set_grading_progress('FullyGraded')
-    ->set_user_id($external_user_id);
+$grade = Packback\Lti1p3\LtiGrade::new()
+    ->setScoreGiven($grade)
+    ->setScoreMaximum(100)
+    ->setTimestamp(date(DateTime::ISO8601))
+    ->setActivityProgress('Completed')
+    ->setGradingProgress('FullyGraded')
+    ->setUserId($external_user_id);
 ```
 
 To send the grade to the platform we can call:
+
 ```php
-$ags->put_grade($grade);
+$ags->putGrade($grade);
 ```
+
 This will put the grade into the default provided lineitem. If no default lineitem exists it will create one.
 
-If you want to send multiple types of grade back, that can be done by specifying an `LTI\LTI_Lineitem`.
+If you want to send multiple types of grade back, that can be done by specifying an `Packback\Lti1p3\LtiLineitem`.
+
 ```php
-$lineitem = LTI\LTI_Lineitem::new()
-    ->set_tag('grade')
-    ->set_score_maximum(100)
-    ->set_label('Grade');
+$lineitem = Packback\Lti1p3\LtiLineitem::new()
+    ->setTag('grade')
+    ->setScoreMaximum(100)
+    ->setLabel('Grade');
 
-$ags->put_grade($grade, $lineitem);
+$ags->putGrade($grade, $lineitem);
 ```
 
 If a lineitem with the same `tag` exists, that lineitem will be used, otherwise a new lineitem will be created.
 
+## Laravel Implementation Guide
```

Comment:
Do we have any idea how other folks have used this package? Is it solely with Laravel or are there more general use cases? This README starts off hitting you with a lot of technical details. Which, may or may make sense, but it's still pretty intimidating for someone with minimal LTI 1.3 experience.

Reply:
No idea. I know there's at least one person using this branch, but have no idea who else might user it or how.

Any recommendations for how to make the readme more accessible to new users?

Reply:
Mhmm, I don't think have any practical suggestions.

It largely depends on target audience and how much effort we want to put into it. What comes to mind is either walking through integrating this with a fresh Laravel app or linking to another open-source repo demoing the already setup library. At least for me, it's hard to see how all of these pieces are tied together at first when all the technical details are laid out without runnable examples. Though, the newly added tests do help out here.